keps/2724-topology-aware-scheduling/README.md

```diff
@@ -663,11 +685,23 @@ For a given PodSet Kueue:
   level. Kueue starts the search from the specified level, but if the PodSet
   does not fit, then it tries higher levels in the hierarchy.
 
-Kueue packs pods on domains starting from the domains with the most free capacity. However, Kueue can operate in two modes when it comes to choosing the last domain if there is more than one capable of accommodating the remaining pods:
-- `MostAllocated` - Kueue chooses the domain with the least available resource that is capable of accommodating all the pods to mitigate resource fragmentation
-- `LeastAllocated` - Kueue chooses the domain that has the most available resources, providing better nodes utilization
+Kueue places pods on domains with different algorithms, depending on the annotation and chosen profile:
+- `MostFreeCapacity` algorithm - Kueue selects as many domains as needed (if it meets user's requirement) starting from the one with the most free capacity;
+- `LeastFreeCapacity` algorithm - Kueue selects as many domains as needed (if it meets user's requirement) starting from the one with the least free capacity;
+- `BestFit` algorithm - Kueue selects as many domains as needed (if it meets user's requirement) starting from the one with the most free capacity, but tries to optimize the last domain to leave as few free resources as possible.
```

Comment:
If the topology is flat (only nodeLabel), what happens?

Reply:
We don't know yet, but only BestFit sounds like a reasonable default.

We may still keep other algorithms, depending on user feedback, under some dedicated configuration (not feature gates), and the user feedback will help us to design the configuration. 

I think we could already open an issue for revisiting the TAS configuration regarding the placement algorithms.

Reply:
Sorry what I wanted to ask is about "but tries to optimize the last domain to leave as few free resources as possible."

Does this "last domain" mean lowest level, right? In that case, this BesiFit algorithm behavior is mostly LeastFreeCapacity in the case of flat topology (only nodeLabel)?

Reply:
No, the last domain means the last domain to choose at any given level. Imagine you want to schedule 7 pods and nodes with free capacity that can fit 3,3,2 and 1 pod. In that case both `BestFit` algorithm and `MostFreeCapacity` algorithm choose first two nodes with 3 available pods. The difference occurs when selecting the last domain:
- `BestFit` will choose a node that can fit 1 pod, as it's enough
- `MostFreeCapacity` will choose a node that can fit 2 pods, as it's the one with most free capacity

Reply:
Clarified, and added example as well

Reply:
Thank you, looks great.